EIPS/eip-4950.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: An ERC721-compatible standard with only two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721
+---
+
+## Abstract
+We define entangled tokens to A and B to the tokens that share a same wallet, so that any transaction with the wallet could be operated by any of them.
+
+## Motivation
+The functionality is similar to the entanglement of two particles but with tokens, in the sense that whatever changes the state of A it affects B. Changes in the state of a token are actuated on a wallet by means of transfer to and from the wallet of whatever fungible and non-fungible tokens.
+
+Initially both minted tokens belong to the contract creator, but they can be transferred normally as any other NFT, so that they may belong to different users. The two users owning tokens can operate with the contract wallets. Therefore, the contract wallet (which may contain balances in several tokens) is “tangled” with both users. If a user makes some transfer, automatically the other user “sees” it.
+
+## Specification
+A tangled token is an NFT (ERC-721) with a modification such that it does not mint only one token at the contract deployment, but it mints two instead, and the token owners have access to the contract wallet.
+
+The constructor should mint the unique token of the contract, and then the mint function should add a restriction to avoid more minting.
+
+Also, a `tokenTransfer` function should be added in order to allow the contract owner to transact with the ERC20 tokens owned by the contract/NFT itself.
+
+## Rationale
+We advocate the entangled tokens have key utility for the value preservation (vreservation) of digital assets, though other further applications are here explained. In the case of value preservation, its importance arises since we'll see a massive migration of value (in the history of mankind) onto the Blockchain along with a new identity forged on blockchain for this value. This value needs to be preserved. So we see how value migrates (1), tokenises (2) and preserves (3) the value of digital assets as NFT as a means of a totally new identity and wealth management for a private and safe assets scrow for all, and redefining the today’s role of consumers towards a more co-creation and co-ownership of locals.

```

Comment:
This section reads more like it should go in "Motivation". The Rationale section is for describing the reasoning behind specific choices in this EIP. An example for this particular EIP could be why you chose to extend EIP-721 instead of EIP-1155.